webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/reports/ComplianceLevel.scala

```diff
@@ -93,6 +93,44 @@ final case class CompliancePercent(
   val compliance = success + repaired + notApplicable + compliant + auditNotApplicable
 }
 
+final case class ComplianceSerializable(
+    applying:                   Option[Double],
+    successNotApplicable:       Option[Double],
+    successAlreadyOK:           Option[Double],
+    successRepaired:            Option[Double],
+    error:                      Option[Double],
+    auditCompliant:             Option[Double],
+    auditNonCompliant:          Option[Double],
+    auditError:                 Option[Double],
+    auditNotApplicable:         Option[Double],
+    unexpectedUnknownComponent: Option[Double],
+    unexpectedMissingComponent: Option[Double],
+    noReport:                   Option[Double],
+    reportsDisabled:            Option[Double],
+    badPolicyMode:              Option[Double]
```

Comment:
the elements are not in the same order than `CompliancePercent`, that seems to be an extremely easy way to have errors. 
For the naming, it seems better to use the event name like you did, but I wonder why we don't use them in CompliancePercent too.